### PR TITLE
Cult Forging Enhancement

### PR DESCRIPTION
**Pull request by Geevies**
* Added cult descriptions to items that can be reforged in a daemon forge.
* Added some more items to be reforged.
* Eldritch voidsuit sprites now have in-built shoes.
* Eldritch voidsuits have been slightly buffed.
* Fixed eldritch helmets not having the correct light colour.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/gamemodes/cult/cultify/obj.dm [hunk 1]
```diff
@@ -12,22 +14,33 @@
...
+	return "As you cast \the [src] into the forge, a great pylon grows beneath your feet, achieving its form in seconds."
 
 /obj/item/stack/material/wood/cultify()
 	return
 
 /obj/item/book/cultify()
-	new /obj/item/book/tome(get_turf(src))
-	..()
+	var/obj/item/book/tome/T = new /obj/item/book/tome(get_turf(src))
+	..(FALSE)
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/cult/cultify/obj.dm:25]: At this point don't add a default value to the cultify proc if you're always overriding it

File: code/game/gamemodes/cult/structures/forge.dm [hunk 2]
```diff
@@ -6,5 +6,13 @@
...
+		var/stored_message = "You cast \the [W] into the forge, where it rapidly changes form. In a flash, you see it reappear on your person."
+		var/cult_item = W.cultify(TRUE)
+		if(isnull(cult_item))
+			to_chat(user, SPAN_WARNING("You get the idea that you can't reforge this."))
+			return
+		if(istype(cult_item, /obj/item))
+			user.put_in_hands(cult_item)
+			to_chat(user, span("cult", stored_message))
+		else if(!isnull(cult_item)) // it didn't return an item
+			to_chat(user, span("cult", cult_item)) // but i still want it to play a message
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/cult/structures/forge.dm:18]: not necessary but should probs make font_cult

File: code/game/atoms.dm [hunk 3]
```diff
@@ -146,6 +146,8 @@
 
 	to_chat(user, "\icon[src] That's [f_name] [suffix]")
 	to_chat(user, desc)
+	if(description_cult && (user.mind?.special_role == "Cultist" || isobserver(src)))
```
Comment (Alberyk) [inline code/game/atoms.dm:149]: Can't this just use description_antag instead?

Comment (Geevies) [reply code/game/atoms.dm:149]: no, wouldn't look good

File: code/game/gamemodes/cult/items/armor.dm [hunk 4]
```diff
@@ -14,29 +14,29 @@
 	flags_inv = HIDEGLOVES|HIDEJUMPSUIT
 
 /obj/item/clothing/head/helmet/space/cult
-	name = "eldritch helmet"
-	desc = "A bulky helmet, bristling with spikes. It looks space proof."
+	name = "eldritch voidsuit helmet"
+	desc = "A bulky armoured voidsuit helmet, bristling with menacing spikes. It looks space proof."
 	icon_state = "cult_helmet"
-	armor = list(melee = 60, bullet = 50, laser = 30, energy = 15, bomb = 30, bio = 30, rad = 30)
+	armor = list(melee = 60, bullet = 50, laser = 45, energy = 15, bomb = 30, bio = 30, rad = 50)
```
Comment (Arrow768) [inline code/game/gamemodes/cult/items/armor.dm:20]: Not a fan of these buffs in combination with the reforging, as reforging cult armour to a space suit should have some downside (which is currently in the form of slightly less protection)

File: code/game/gamemodes/cult/items/armor.dm [hunk 5]
```diff
@@ -14,29 +14,29 @@
...
 	icon_state = "cult_armour"
 	item_state = "cult_armour"
-	desc = "A bulky suit of armour, bristling with spikes. It looks space proof."
-	w_class = 3
+	desc = "A bulky armoured voidsuit, bristling with menacing spikes. It looks space proof."
+	w_class = ITEMSIZE_NORMAL
 	allowed = list(/obj/item/book/tome, /obj/item/melee/cultblade, /obj/item/tank, /obj/item/device/suit_cooling_unit)
 	slowdown = 1
-	armor = list(melee = 60, bullet = 50, laser = 30, energy = 15, bomb = 30, bio = 30, rad = 30)
+	armor = list(melee = 60, bullet = 50, laser = 45, energy = 15, bomb = 30, bio = 30, rad = 50)
```
Comment (Arrow768) [inline code/game/gamemodes/cult/items/armor.dm:36]: Same here

File: code/game/gamemodes/cult/items/armor.dm (hunk 4 above)
Comment (Geevies) [inline code/game/gamemodes/cult/items/armor.dm:20]:
```suggestion
	armor = list(melee = 60, bullet = 50, laser = 35, energy = 15, bomb = 30, bio = 30, rad = 30)
```

File: code/game/gamemodes/cult/items/armor.dm (hunk 5 above)
Comment (Geevies) [inline code/game/gamemodes/cult/items/armor.dm:36]:
```suggestion
	armor = list(melee = 60, bullet = 50, laser = 45, energy = 15, bomb = 30, bio = 30, rad = 30)
```

File: code/game/gamemodes/cult/items/armor.dm [hunk 6]
```diff
@@ -14,29 +14,29 @@
...
 	icon_state = "cult_armour"
 	item_state = "cult_armour"
-	desc = "A bulky suit of armour, bristling with spikes. It looks space proof."
-	w_class = 3
+	desc = "A bulky armoured voidsuit, bristling with menacing spikes. It looks space proof."
+	w_class = ITEMSIZE_NORMAL
 	allowed = list(/obj/item/book/tome, /obj/item/melee/cultblade, /obj/item/tank, /obj/item/device/suit_cooling_unit)
 	slowdown = 1
-	armor = list(melee = 60, bullet = 50, laser = 30, energy = 15, bomb = 30, bio = 30, rad = 30)
+	armor = list(melee = 60, bullet = 50, laser = 45, energy = 15, bomb = 30, bio = 30, rad = 30)
```
Comment (Arrow768) [inline code/game/gamemodes/cult/items/armor.dm:36]:
```suggestion
	armor = list(melee = 60, bullet = 50, laser = 30, energy = 15, bomb = 30, bio = 30, rad = 30)
```

File: code/game/gamemodes/cult/items/armor.dm [hunk 7]
```diff
@@ -14,29 +14,29 @@
 	flags_inv = HIDEGLOVES|HIDEJUMPSUIT
 
 /obj/item/clothing/head/helmet/space/cult
-	name = "eldritch helmet"
-	desc = "A bulky helmet, bristling with spikes. It looks space proof."
+	name = "eldritch voidsuit helmet"
+	desc = "A bulky armoured voidsuit helmet, bristling with menacing spikes. It looks space proof."
 	icon_state = "cult_helmet"
-	armor = list(melee = 60, bullet = 50, laser = 30, energy = 15, bomb = 30, bio = 30, rad = 30)
+	armor = list(melee = 60, bullet = 50, laser = 35, energy = 15, bomb = 30, bio = 30, rad = 30)
```
Comment (Arrow768) [inline code/game/gamemodes/cult/items/armor.dm:20]:
```suggestion
	armor = list(melee = 60, bullet = 50, laser = 30, energy = 15, bomb = 30, bio = 30, rad = 30)
```

File: html/changelogs/geeves-cult_forging.yml [hunk 8]
```diff
@@ -0,0 +1,10 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - rscadd: "Added cult descriptions to items that can be reforged in a daemon forge."
+  - rscadd: "Added some more items to be reforged."
+  - rscadd: "Eldritch voidsuit sprites now have in-built shoes."
+  - rscadd: "Eldritch voidsuits have been slightly buffed."
```
Comment (Arrow768) [inline html/changelogs/geeves-cult_forging.yml:9]:
```suggestion
```

**Pull request merged**